pkg/ebpf/c/maps.h

```diff
@@ -2,10 +2,12 @@
 #define __MAPS_H__
 
 #include <vmlinux.h>
-#include <missing_definitions.h>
+#include <vmlinux_flavors.h>
+#include <vmlinux_missing.h>
```

Comment:
Are these headers needed here?

Reply:
ditto (tried to create a pattern, can remove it).